src/staff-handbook/pay-pension-and-benefits/pension.md

```diff
@@ -3,15 +3,10 @@ title: Pension
 related_order: 2
 last_reviewed_at: ""
 ---
+dxw provides a Pension Salary Exchange (PSE) scheme which is operated by Aviva. In a PSE scheme, contributions are deducted from your gross salary, resulting in a saving on tax and National Insurance Contributions (NICs). 
 
-dxw provides a pension which is operated by Aviva. Once you have passed your 3-month check in, you will be auto enrolled into the scheme, receive details by
-post of the necessary employer/employee contributions and have the option to set
-your percentage or opt out.
+In a PSE scheme, dxw will pay all pension contributions into your pension scheme, meaning a higher amount of employer contributions. The increase in employer contributions is then offset by an equivalent reduction in gross salary. dxw takes the salary you have exchanged and pays that straight into your pension fund.
```

Comment:
```suggestion
Within the PSE scheme, dxw pays all pension contributions into your pension scheme, meaning a higher amount of employer contributions. The increase in employer contributions is then offset by an equivalent reduction in gross salary. dxw takes the salary you have exchanged and pays that straight into your pension fund.
```

Reply:
Okay - this is just making the tense consistent within the paragraph (c.f. "is then offset" not "will then be offset" and "dxw takes" not "dxw will take"), but it's a minor thing